### DISCUSSION TITLE
fzcd : Does this plugin respect .gitignore ?

### DISCUSSION THREAD

**Discussion by rock-tran** [jarun/nnn#1676] [Q&A] [answered]
As I try, this plugin seems respect .gitignore. May I run this plugin without respecting the .gitignore ?

**Comment by amalgame21** [comment, marked as answer]
What is the output of `echo $FZF_DEFAULT_COMMAND` in your terminal ?

https://github.com/junegunn/fzf/issues/1625#issuecomment-507674106

**Reply by rock-tran** [reply]
@amalgame21 It's here
```
fd --type f --hidden --exclude .git
```

**Reply by rock-tran** [reply]
@amalgame21 Thanks. I only need to export FZF_DEFAULT_COMMAND="fd --type f -HI --exclude .git" to search .gitignore files.

**Reply by amalgame21** [reply]
@rock-tran ya, or you can just put `export FZF_DEFAULT_COMMAND="fd --type f -HI --exclude .git"` in the plugin (near the beginning of the file) to make only this plugin respect the .gitignore but not globally.
